### PR TITLE
Italian translation (Update)

### PR DESCRIPTION
**Pull request by astrone**
There is an error in Privacy Settings with italian translation. Here a screenshot: https://db.tt/IPw7eaf5 
I have fixed it however i could not find the new strings "Tabs" , "Reader mode" etc.. hope to see them soon!

### CODE REVIEW

File: res/values-it/strings.xml [hunk 1]
```diff
@@ -38,7 +39,7 @@
     <string name="search_hint">Cerca</string>
     <string name="wideViewPort">Use wide viewport</string>
     <string name="overViewMode">Load pages in overview mode</string>
-    <string name="restore">Restore lost tabs on start</string>
+    <string name="restore">Ricarica le schede perse all'avvio</string>
```
Comment (karolba) [inline res/values-it/strings.xml]: Shouldn't `'` be `\'`?

File: res/values-it/strings.xml [hunk 2]
```diff
@@ -101,5 +100,26 @@
...
     <string name="bookmark_settings">Impostazioni Segnalibri</string>
     <string name="title_chooser">Seleziona un file</string>
-
+    <string name="block">Blocca le immagini</string>
+    <string name="install_orbot">Installa Orbot per attivare questa funzione.</string>
+    <string name="settings_general">Impostazioni Generali</string>
+    <string name="settings_display">Impostazioni Display</string>
+    <string name="settings_privacy">Privacy</string>
+    <string name="settings_about">Informazioni</string>
+    <string name="settings_about_explain">Informazioni sull'autore, versione e license.</string>
```
Comment (karolba) [inline res/values-it/strings.xml]: The same here. Shouldn't `'` be `\'`?

File: res/values-it/strings.xml (hunk 1 above)
Comment (astrone) [inline res/values-it/strings.xml]: Wops, yeah you're right.

**Comment by karolba** [comment]
Youm

**Comment by karolba** [comment]
You should update your branch, the changes will appear here. And use `\`, not `/`.

**Comment by kuc** [comment]
I also suggest to use `git commit --amend` and then `git push --force` to clean up this PR after fixing all problems.

**Comment by karolba** [comment]
It's from `astrone:patch-2`, GitHub creates branches named like that when you press the edit button from the web interface.

**Comment by astrone** [comment]
Thanks guys! Finally fixed it, everything should be right now as the build passed.

**Pull request merged**